test/whl-scale-tests/deploy-scale-test-suite.ps1

```diff
@@ -0,0 +1,214 @@
+param(
+    [guid] [Parameter(Mandatory = $true)] $SubscriptionId,
+    [string] [Parameter(Mandatory = $true)] $Location,
+    [string] [Parameter(Mandatory = $true)] $GenevaAccountName,
+    [string] [Parameter(Mandatory = $true)] $GenevaLogAccountNamespace,
+    [string] [Parameter(Mandatory = $true)] $CrashDumpConfigVersion,
+    [string] [Parameter(Mandatory = $true)] $ETWConfigVersion,
+    [string] [Parameter(Mandatory = $true)] $EventLogConfigVersion,
+    [string] [Parameter(Mandatory = $true)] $TextLogConfigVersion
+)
+
+$orignalPath = Get-Location
+Set-Location -Path $PSScriptRoot
+. .\common.ps1
+
+$genevaEnvironment = "DiagnosticsProd"
+$resourceGroupName = [Environment]::UserName + "scaletest"
+$aksClusterName = $resourceGroupName + "aks"
+$acrName = $resourceGroupName + "acr"
+$acrUri = $acrName + ".azurecr.io"
+
+# Login using your microsoft accout
+Write-Host "Login with your Microsoft account"
+az login
+
+# Set subscription
+Write-Host "Setting az account to given Subscription"
+az account set --subscription $SubscriptionId
+
+#Use Windows Engine on Docker
+Write-Host "Setting Docker to utilize Windows Engine"
+Start-Process -filePath "Docker Desktop.exe" -WorkingDirectory "C:\Program Files\Docker\Docker"
+Start-Sleep -Duration (New-TimeSpan -Seconds 60)
+Start-Process -filePath "DockerCli.exe" -WorkingDirectory "C:\Program Files\Docker\Docker" -ArgumentList "-SwitchWindowsEngine"
+
+#Login into ACR
+Write-Host "Logining into ACR"
+az acr login --name $acrName
+
+#Create latest WHL Container Image
+$imageName = $acrUri + "/latestwhl:$(Get-Date -Format MMdd)"
+Write-Host "Moving working directory to ..\..\kubernetes\windows\hostlogs"
+Set-Location "..\..\kubernetes\windows\hostlogs"
+
+$filepath = ".\build-and-publish-docker-image.ps1"
+$dockerCommandArguments = "imageTag  ."
+
+$dockerCommandHashTable = @{
+    $dockerCommandArguments = $dockerCommandArguments + " --network `"Default Switch`"";

```

Comment:
There's not always a "Default Switch" network so this can cause an error
> network Default Switch not found

Reply:
Was able to repro this issue. Looking for another route to fix this.

Reply:
Just removed it as it was causing more harm than good